package.json

```diff
@@ -16,17 +16,17 @@
     "test": "test"
   },
   "scripts": {
-    "test": "echo \"no tests :-(\" && echo 0",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
I use the `exit 0` so that my deploy scripts don't cancel the deploy because of failing tests